src/Provider/Nominatim/Nominatim.php

```diff
@@ -160,6 +164,7 @@ private function xmlResultToArray(\DOMElement $resultNode, \DOMElement $addressN
             $countryCode = strtoupper($countryCode);
         }
         $builder->setCountryCode($countryCode);
+
```

Comment:
Why what this blank line added?

Reply:
Same as previous comment, seemd to make more sense to create a "block" with country code process ;
But again, really don't mind putting that back as it was :)

Reply:
Okey. I just saw a weird diff. This is fine